README.md

```diff
@@ -954,6 +954,10 @@ jobs:
           server: js.example.net
 ```
 
+### Kubernetes JetStream Controller
+
+The JetStream controllers allows you to manage NATS JetStream Streams and Consumers via K8S CRDs. You can find more info on how to deploy and usage [here](https://github.com/nats-io/nack#getting-started).
+
```

Comment:
Maybe just add a small example like the GH and TF ones have so people can get a quick look